backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java

```diff
@@ -1019,7 +1058,7 @@ void fetchTestEventsResults_getTestEventsResults_NPlusOne() {
     _dataFactory.createTestEvent(p, facility);
     _dataFactory.createTestEvent(p, facility);
 
-    // count queries again and make queries made didn't increase
+    // count queries again and make sure queries made didn't increase
```

Comment:
This test is now failing, presumably because we eagerly load Results from the database. I also tried lazy loading Results and initializing directly in `TestOrderService`, but it still failed. Any ideas on how to fix these?